Dockerfile

```diff
@@ -14,7 +14,12 @@ RUN apt-get update \
         sudo \
         unzip \
         zip \
-    && curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+    && apt-get update \
+    && apt-get install -y ca-certificates curl gnupg \
```

Comment:
We install `ca-certificates` and `curl` already above, so could remove those, and then at that point I'd also just put `gnupg` in that same `apt-get install` block.